update.js

```diff
@@ -0,0 +1,124 @@
+
+async function main() {
+    /**
+     * Connection URI. Update <username>, <password>, and <your-cluster-url> to reflect your cluster.
+     * See http://bit.ly/NodeDocs_lauren for more details
+     */
+    const uri = "mongodb+srv://<username>:<password>@<your-cluster-url>/test?retryWrites=true&w=majority";
+
+    /**
+     * The Mongo Client you will use to interact with your database
+     * See bit.ly/Node_MongoClient for more details
+     */
+    const client = new MongoClient(uri);
+
+    try {
+        // Connect to the MongoDB cluster
+        await client.connect();
+
+        // Make the appropriate DB calls
+
+        // UPDATE
+        // Print the Infinite Views listing
+        await findListingByName(client, "Infinite Views");
+        // Update the Infinite Views listing to have 6 bedrooms and 8 beds 
+        await updateListingByName(client, "Infinite Views", { bedrooms: 6, beds: 8 });
+        // Print the updated Infinite Views listing
+        await findListingByName(client, "Infinite Views");
+
+        // UPSERT
+        // Check if a listing named Cozy Cottage is in the db
+        await findListingByName(client, "Cozy Cottage");
+        // Upsert the Cozy Cottage listing
+        await upsertListingByName(client, "Cozy Cottage", { name: "Cozy Cottage", bedrooms: 2, bathrooms: 1 });
+        // Print the details of the Cozy Cottage listing
+        await findListingByName(client, "Cozy Cottage");
+        // Upsert the Cozy Cottage listing
+        await upsertListingByName(client, "Cozy Cottage", { beds: 2 });
+        // Print the details of the Cozy Cottage listing
+        await findListingByName(client, "Cozy Cottage");
+
+        // UPDATE MANY
+        // Update all listings so they have a property type
+        await updateAllListingsToHavePropertyType(client);
+        // Print the details of the Cozy Cottage listing that should now have a property type
+        await findListingByName(client, "Cozy Cottage");
+
+    } finally {
+        // Close the connection to the MongoDB cluster
+        await client.close();
+    }
+}
+
+main().catch(console.error);
+
+/**
+ * Update an Airbnb listing with the given name
+ * Note: If more than one listing has the same name, only the first listing the database finds will be updated.
+ *       It's best to use updateOne when querying on fields that are guaranteed to be unique.
+ * @param {MongoClient} client A MongoClient that is connected to a cluster with the sample_airbnb database
+ * @param {string} nameOfListing The name of the listing you want to update
+ * @param {object} updatedListing An object containing all of the properties to be updated for the given listing
+ */
+async function updateListingByName(client, nameOfListing, updatedListing) {
+    // See http://bit.ly/Node_updateOne for the updateOne() docs
+    result = await client.db("sample_airbnb").collection("listingsAndReviews").updateOne({ name: nameOfListing }, { $set: updatedListing });
```

Comment:
`const result =`

Reply:
You've called me on this before.  Thanks for catching them!  I'll try to catch them in the future.